static/app/gettingStartedDocs/unreal/unreal.tsx

```diff
@@ -30,6 +30,25 @@ void Verify()
     SentrySubsystem->CaptureMessage(TEXT("Capture message"));
 }`;
 
+const getSettingsConfigureSnippet = (params: Params) => `
+#include "SentrySubsystem.h"
+
+FConfigureSettingsDelegate OnConfigureSettings;
+OnConfigureSettings.BindDynamic(this, &UMyGameInstance::ConfigureSentrySettings);
+
+void UMyGameInstance::ConfigureSentrySettings(USentrySettings* Settings)
+{
+    Settings->Dsn = TEXT("${params.dsn.public}");
+
+    // Add data like request headers, user ip address, device name, etc.
```

Comment:
There's no request header in Unreal though. What would be the right comment?
```suggestion
    // Add data like ip address and device name
```

We should link to the docs page we're creating where we list what this is

Reply:
Good point, I've just added "Data Collected" page to docs (https://docs.sentry.io/platforms/unreal/data-management/data-collected/) and will link it here as well